packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -67,6 +67,8 @@ class AppSizeController {
   static const identicalFilesError =
       'Failed to load diff: OLD and NEW files are identical.';
 
+  static const deferredRootName = 'ArtificialRoot';
```

Comment:
Let's rename this to `artificialRootNodeName`. Let's also add another constant `mainNodeName` which is equal to `Main`